deploy/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM maven:3.6.1-jdk-11 as MAVEN
+
+COPY . /fta-fmea
+WORKDIR /fta-fmea/ontology-generator
+RUN mvn clean install
+
+FROM gradle:6.4-jdk11 as GRADLE
+COPY . /fta-fmea
+WORKDIR /fta-fmea
+
+COPY --from=MAVEN /fta-fmea/src/main/generated/cz/cvut/kbss/analysis/util/Vocabulary.java \
+                   ./src/main/generated/cz/cvut/kbss/analysis/util/Vocabulary.java
+
+RUN ./gradlew clean war
+
+FROM tomcat:8-jdk11
+
+COPY --from=GRADLE /fta-fmea/build/libs/fta-fmea-0.0.1-SNAPSHOT.war /usr/local/tomcat/webapps/fta-fmea.war
```

```suggestion
COPY --from=GRADLE /fta-fmea/build/libs/fta-fmea-*.war /usr/local/tomcat/webapps/fta-fmea.war
```

@Matthew-Kulich this will not work if the version changes. 



Btw, is it needed to do such complicated copying and even packaging tomcat? Can't we just use spring boot for that  (I saw it in project) ?
See some ideas in gradle project here https://github.com/opendata-mvcr/sgov/blob/main/Dockerfile